ynput/ayon-unreal
# Editorial_pkg: publishing sequence into OTIO timeline with rendered media

## Changelog Description
Implement a sequence-based workflow for publishing an editorial package in Unreal. This workflow should publish the OTIO timeline and an intermediate render in MP4 format.

Resolve #131, 
Resolve https://github.com/ynput/ayon-unreal/issues/125
Resolve https://github.com/ynput/ayon-unreal/issues/124

- [x] otio representation implementation
- [x] Has `unreal_export.py` edited for the otio implementation
- [x] Edit rendering.py which can be used for both rendering and editorial intermediate rendering

## Additional info
-It may need multiple testing to get the correct frame ranges
@jakubjezek001 this is the zipped file with otio and the clip: 
[il_UE_editorial_pkgMain_v006.zip](https://github.com/user-attachments/files/22137977/il_UE_editorial_pkgMain_v006.zip)

## Testing notes:
1. Select level sequence which includes MovieShotCutTrack/CameraCutTrack with multiple sections
2. Create Editorial Package Instance with it
3. Ayon -> Render
4. Wait to render out all the sequences until finished
5.  Ayon -> Publish
6. It should publish successfully with mp4 and otio
7. Load otio in hiero/resolve
